app/controllers/admin/community_transactions_controller.rb

```diff
@@ -54,7 +54,6 @@ def index
 
     render("index",
       { locals: {
-        show_status_and_sum: @current_community.payments_in_use?,
```

Comment:
It is still used in a few places. 

The logic is definitely wrong for asking whether payments are in use in this marketplace. RIght way to define that would be to see all the Order Types if any of them uses process != free.

The logic is right, if the question is "does this community have any payment gateways configured". And that question is also a "legacy" question, since all our new trials will have PayPal enabled even if they don't have any Order Type with Online payments. The only reason why we remove PayPal from a community is that they want to use currency that is not supported by PayPal.